geomstats/backend/numpy_linalg.py

```diff
@@ -37,3 +37,6 @@ def eig(*args, **kwargs):
 
 def exp(*args, **kwargs):
     return np.exp(*args, **kwargs)
```

Comment:
Hm I'm not sure why this is in numpy_linalg and not numpy. @johmathe, an idea?

Reply:
Removed